pkg/helpers/homebrew.go

```diff
@@ -0,0 +1,72 @@
+package helpers
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/devbuddy/devbuddy/pkg/utils"
+)
+
+type caskroom struct {
+	prefix string
+}
+
+type cellar struct {
+	prefix string
+}
+
+// Homebrew represent an homebrew installation
+type Homebrew struct {
+	caskroom *caskroom
+	cellar   *cellar
+}
+
+// NewHomebrew is returning a new Cellar
+func NewHomebrew() *Homebrew {
+	prefix := "/usr/local"
+
+	return &Homebrew{
+		cellar:   &cellar{prefix: prefix},
+		caskroom: &caskroom{prefix: prefix},
+	}
+}
+
+// NewHomebrewWithPrefix is returning a new Cellar at prefix
+func NewHomebrewWithPrefix(prefix string) *Homebrew {
+	return &Homebrew{
+		cellar:   &cellar{prefix: prefix},
+		caskroom: &caskroom{prefix: prefix},
+	}
+}
+
+// IsInstalled returns true if `pkg` is installed in cellar or in caskroom
+func (h *Homebrew) IsInstalled(formula string) (installed bool) {
+	path := buildFormulaPath(formula)
+
+	return h.cellar.IsInstalled(path) || h.caskroom.IsInstalled(path)
+}
+
+func buildFormulaPath(filename string) string {
```

Comment:
The logic doesn't seem so obvious, I feel like we should explain what happens here.